src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="/src/images/logo.png"
+          alt="MOYO"
+          class="logo_image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">APPLE</a>
+
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">SAmSUNG</a>
+          </li>
+          <li class="nav_item">
```

Comment:
Add the blank line between the all links
```suggestion
          </li>
          
          <li class="nav_item">
```